ui/admin/app/components/form/credential-library/index.hbs

```diff
@@ -74,6 +74,24 @@
     {{/each}}
   </form.radioGroup>
 
+  <Rose::Form::Fieldset>
```

Comment:
Since named block `form` is already available, this can be converted to `<form.fieldset>`.

Reply:
addressed!